sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -305,6 +323,20 @@ static const struct snd_soc_acpi_link_adr tgl_3_in_1_sdca[] = {
 	{}
 };
 
+static const struct snd_soc_acpi_link_adr tgl_workaround_hp[] = {
+	{
+		.mask = BIT(0),
+		.num_adr = ARRAY_SIZE(workaround_rt711_0_adr),
+		.adr_d = workaround_rt711_0_adr,
+	},
+	{
+		.mask = BIT(1),
+		.num_adr = ARRAY_SIZE(workaround_rt1308_1_single_adr),
+		.adr_d = rt1308_1_single_adr,
```

Comment:
is it workaround_rt1308_1_single_adr ?

Reply:
yes, I didn't touch tgl for now, this is needed as well.

Reply:
fixed now